### PR TITLE
a lot of medical changes

### PR DESCRIPTION
**Pull request by pudl**
-removes an extra nanomed i added to virology by accident long ago
-adds a spare oxygen kit to cryo
-makes iv drips no longer dense
-changes a stupid fluff message on iv drips
-improves first aid names and descriptions
-removes gauze from the primary first aid kid and instead puts it in the brute one
-fixes incorrect pill bottle names, again my mistake
-comments out the possibility for oxygen closets to be plain empty or not appear at all, oxygen kit chance increased
-adds colors and better descriptions to most chemicals, as well as a lot of mix messages
-slightly buffs salglu silver sulf
-changes many chemistry related fluff messages
-changes the max amount of volume on patches to be 40
-makes pre spawned patches only start with 20
-adds what chemical is inside of patches to the description
-adds a few more medkits around
-adds a "general storage" room
-puts some improvised gauze in maintenance
-gives improvised gauze its own unique sprite
-adds medical patches to general first aid
-removes an extraneous first aid from the first aid crate
-ups the nanomed from 8 gauze to 10
-changes gauze description
-fixes an extra /
-renames improvised gauze to "strips of cloth"
-i think that's it

### CODE REVIEW

**Comment by Cheridan** [comment]
Good changes.

**Comment by WJohn** [comment]
I'm not a huge fan of the total removal of good descriptions in the chem master analyze messages. There's a lot of RNG healing or toxins that you can't possibly tell any numbers from without it.

**Comment by pudl** [comment]
read the wiki, wjohn

**Comment by Steelpoint** [comment]
I'm almost finished with a new and updated chem wiki page which should allow more transparency.

https://tgstation13.org/wiki/User:Steelpoint

**Comment by GunHog** [comment]
The best ingame descriptions are a mix of fluff and information. Do not remove the information entirely. Mention the effect so a person can know what the chem does *without feeling the need to need a wiki*. The point of the description is to tell you what it does. There is no point in having it at all if you intentionally withhold information from the player.

**Comment by MrPerson** [comment]
What GunHog said. "It's on the wiki" is an unaccpetable bullshit cop-out

**Comment by pudl** [comment]
every single desc sans a few joke chems or drugs give a description of what the chemical does. i based these descriptions off our old ones, so i doubt that anyone would be mad about these
on top of that, the only way to see these descs is the analyzer upgrade from rnd. its nearly pointless

**Comment by Jordie0608** [comment]
Changelog

**Comment by Dorsisdwarf** [comment]
Puddle you went too faaaar. It's fine to say "Brain damage" or "Anti-histamine" or "Is only effective on critical patients" so long as it's not "Takes 3 cycles to make you honk for twelve cycles, then heals all your BRUTE"

Also you can see these descriptions with the (?) button in the chem dispenser.

**Comment by Fox-McCloud** [comment]
:+1: to the descriptions

File: code/modules/reagents/Chemistry-Goon-Medicine.dm [hunk 1]
```diff
@@ -34,52 +34,52 @@ datum/reagent/silver_sulfadiazine/on_mob_life(var/mob/living/M as mob)
...
+	color = "#696969" // rgb: 200, 165, 220
+	metabolization_rate = 0.15
 
 datum/reagent/salglu_solution/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
 	if(prob(33))
-		M.adjustBruteLoss(-1*REM)
-		M.adjustFireLoss(-1*REM)
+		M.adjustBruteLoss(-2*REM)
+		M.adjustFireLoss(-2*REM)
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Goon-Medicine.dm]: Hey, I nerfed this for being overpowered, revert this shit now.

File: code/modules/reagents/reagent_containers/patch.dm [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	icon_state = "bandaid"
 	item_state = "bandaid"
 	possible_transfer_amounts = null
-	volume = 50
+	volume = 40
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: Stop doing this shit, and especially stop "forgetting" to mention it in the description.

**Comment by Iamgoofball** [comment]
I like the descriptions but you keep trying to fuck with other shit I'm already fucking with.

**Comment by pudl** [comment]
QUIT TOUCHIN MUH CODEBABBY

File: code/modules/reagents/reagent_containers/patch.dm (hunk 2 above)
Comment (pudl) [inline code/modules/reagents/reagent_containers/patch.dm]: no

File: code/modules/reagents/Chemistry-Goon-Medicine.dm (hunk 1 above)
Comment (pudl) [inline code/modules/reagents/Chemistry-Goon-Medicine.dm]: alright

File: code/modules/reagents/reagent_containers/patch.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: We talked, :+1: to this change

**Comment by PKPenguin321** [comment]
i dont really see why gauze cant be in the standard medkits as well as the brute ones, there's not really any issue with it

**Comment by pudl** [comment]
pap himself said that medkits need to be far more specialized, him putting gauze in normal medkits was hypocrisy.

**Comment by PKPenguin321** [comment]
well do as you will i guess but at least try to make gauze a little more common somehow, i fear this will make it seen even less

**Comment by pudl** [comment]
you can rip up bedsheets for improvised gauze

**Comment by PKPenguin321** [comment]
bedsheets are a finite material though, plus the ghetto version shouldn't be the primary method of stopping bleeding
again, you could just put gauze in both kits and i dont think anyone would mind, but do as you will

**Comment by pudl** [comment]
@PKPenguin321 i made gauze a lot more common

**Comment by pudl** [comment]
its only a matter of time until merge conflicts

**Comment by pudl** [comment]
FUCKING MERGE CONFLICTS

**Comment by pudl** [comment]
ILL JUST REDO THIS LATER THEN
6 FUCKING VERSIONS

**Pull request closed without merging**